.github/workflows/download_pipeline.yml

```diff
@@ -0,0 +1,134 @@
+name: Test successful pipeline download with 'nf-core pipelines download'
+
+# Run the workflow when:
+#  - dispatched manually
+#  - when a PR is opened or reopened to main/master branch
+#  - the head branch of the pull request is updated, i.e. if fixes for a release are pushed last minute to dev.
+on:
+  workflow_dispatch:
+    inputs:
+      testbranch:
+        description: "The specific branch you wish to utilize for the test execution of nf-core pipelines download."
+        required: true
+        default: "dev"
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+    branches:
+      - main
+      - master
+  pull_request_target:
+    branches:
+      - main
+      - master
```

Comment:
```suggestion
```
fixed in dev version of template and just creates problems once merged, so better to already remove it now 🙂